### ISSUE TITLE
Documentation about how to create dosbox shortcuts

### ISSUE THREAD

**Issue by es20490446e** [dosbox-staging/dosbox-staging#846]
The official dosbox documentation explains how to create shortcuts, but nothing about the format needed for those shortcuts to run commands like imgmount. This is a mayor pain point, as the format is very non obvious.

I brought this issue years ago to the official dosbox forum, but nobody did a thing about it.

So I suggest putting that somewhere on dosbox-staging, in a noticeable place like the readme. Something simple like this:

> Create a shortcut to:
> "[pathToDosboxExecutable]" -c "mount c [pathToGameFolder]" -c "imgmount d [listOfPathsToDiskImages] -t iso" [gameExecutableName] -fullscreen -exit
> 
> During the game, if more than one disk is used, you can change among them with:
> Control + F4

**Comment by nemo93** [comment]
@es20490446e hi. Feel free to participate to the [wiki](https://github.com/dosbox-staging/dosbox-staging/wiki) anytime.

**Comment by es20490446e** [comment]
I will ;)

**Comment by es20490446e** [comment]
Documentation added to: [Imgmount](https://github.com/dosbox-staging/dosbox-staging/wiki/Imgmount)

**Comment by es20490446e** [comment]
@nemo93 @kcgen I see you reverted ALL the changes I made to the wiki, without comments.

I thought you wanted me to made those changes. Are you sure you don't want to write all the documentation by yourselves?

See that I only contribute to projects that know how to be flexible to other people's ideas. Otherwise they aren't really prepare to work along with others, and specially with me. I like to try new approaches all the time.

Bear with me that I wrote the official documentation of important projects, like eMule or Ubuntu One Hundred Papercuts. The quality procedures for debugging the hole operating system.

If you cannot handle my contributions, those of a person that is used to do it, you would likely find more trouble with other people.

**Comment by kcgen** [comment]
@es20490446e  - sorry for not replying here. I wasn't able to find the thread.  

But yes, I reverted the imgmount changes because it was an overall reduction in quality/documentation versus the original.  

All of your other pages regarding the shortcut are left as is, because those are new additions. 

Your edits still edit in the version history.  

I believe the content will be improved if you can insert the shortcut mounting examples in with the exist imgmount page.

I tried to find you on discord however couldn't find your handle - as I wanted to mention it.  (Unfortunately, there is no other communication mechanism here)

**Comment by kcgen** [comment]
@es20490446e ,

You have contributed excellent ideas and recommendations - it is all appreciated, as was your work to inform users how to mount images.  Thank you. 

The only concern was specifically that the entire imgmount wiki article was replaced/removed (and yes, I felt the content was a reduction in overall quality). I believe that when you ammend the article - it will be as good as it currently is plus will have your shortcut improvements: so it will be an overall improvement. 

Of course, that's my opinion: if other readers wish to disagree then we can revert the revert :-)

**Comment by kcgen** [comment]
All edits can be fetched here, so content can be copy and pasted from your prior edit into the existing page: <https://github.com/dosbox-staging/dosbox-staging/wiki/Imgmount/_history>

**Comment by es20490446e** [comment]
Sorry, I made too many assumptions.

Perhaps the shortcuts documentation would be better in its own page, as having the exact target to type could be interesting despite of not using imagemount.

What do you think?

**Comment by kcgen** [comment]
Yes, I think that would let you explore it thoroughly with tips for all the common OSes like windows, macOS, and Linux - and provide some screenshots too (especially to help windows users, as those shortcuts can be tricky to get right, when fielname contain spaces). 

Then in the imgmount page, a one-on-one reference could point users to the page, "Image mounts can be included in desktop launch shortcuts - see here".

**Comment by es20490446e** [comment]
I don't really have Windows to take those screen-shots. For instance I don't need to create those shortcuts myself since I have it automated through [exeCute](https://gitlab.com/es20490446e/exeCute), which also selects automatically the right emulator for every kind of exe.

I just provided that [info](https://archive.org/details/running-dos-games) as a courtesy to the people downloading the games I put into archive.org.

Also I somehow skipped the screen-shots purposely, so for the documentation never gets obsolete. I assumed that most people on Windows know how to create shortcuts, or there's an easy to find tutorial on how to do so. So the tricky part is the exact commands to put as shortcut target.

I would write the most basic documentation. After that we can decide if we want to add more.

**Comment by kcgen** [comment]
Sounds good @es20490446e!

**Comment by es20490446e** [comment]
[Proposal](https://github.com/dosbox-staging/dosbox-staging/wiki/Shortcuts)

**Comment by nemo93** [comment]
I'd be against creating a new page for discussing shortcuts. This is a good fit for the [Tips & Tricks](https://github.com/dosbox-staging/dosbox-staging/wiki/Tips-&-Tricks#shortcuts) page that I revived thanks to your former contribution @es20490446e. Shortcuts can be confusing to new users (quotes are important and if place at the wrong location things won't work). That's why the current [Get Started](https://github.com/dosbox-staging/dosbox-staging/wiki/Get-Started) article steer them to using short `.conf` files only to configure and run applications.

Comments about your proposal:
* get rid of the Unix/Linux section. Users of that OS know how to do that.
* same for Windows section. Just refer the tutorial page in a single sentence.
* what matters with shortcuts are examples: feel free to take some real life examples (eg. with Doom or your favourite games)
* each example must illustrate a specific case that you have to briefly described
* use the triple quote ``` for code lines instead of single `.
* changing disk is out of scope of your section and it's already available [here](https://github.com/dosbox-staging/dosbox-staging/wiki/DOS-commands#IMGMOUNT), [there](https://github.com/dosbox-staging/dosbox-staging/wiki/FAQ#ctrlf4) and [there](https://github.com/dosbox-staging/dosbox-staging/wiki/Special-Keys#default-binding).

Your previous content has been edited as it was not fitting the current layout and articles organization. Also you completely rewrote an article without consulting. I suggest you ask first before doing that as in that case your content was inferior in terms of quality. This will limit frustration on both sides. You can see that for all the other pages, collaboration is working fine.

Editing happened and will happen again as the objective is to keep things as clear and tidy as possible. Hence why the number of main entries on the sidebar is kept to a minimum. New entry is only for subject that really requires thorough explanations, lenghty examples, etc. Users will need to access information in a blink of an eye. Having too many entries will discourage people from searching.

Rules to follow:
1. content is written with new users in mind
2. examples must be clear and backed with data or figures/screenshots
3. keep things tidy (content sorted, short sentences and headlines, etc)

It's an open Wiki and it's really important it stays that way. Collaborating to an open-source project is great and it's good you decided to spend time on Staging. Thanks for that.

**Comment by es20490446e** [comment]
@nemo93 Which means you digress with everything, which suggests me you haven't thought about anything thoroughly enough.

**Comment by es20490446e** [comment]
What's the point of discussing how to create shortcuts if you don't even think the user should be creating them?

For instance I digress that the user should find it easier to create config files than shortcuts.

By the way I think DosBox should be able to figure everything out by itself eventually, instead of delegating the dirty stuff to the user.

**Comment by kcgen** [comment]
@nemo93 , that's a very clear and actionable guideline for contributing to the Wiki. I think that, itself, deserves front-placement in the wiki. 

Please feel free to get rid of the huge image for the default starting page - it's not helping users find or contribute information. 

I would rather the front page start include a short intro and links to the most popular and high-value topics - as you see fit.  Same with this contributing guideline.  (Please treat this just as a ticket from a happy user: you're the lead author of the Wiki and defer to your judgement 👍 )

**Comment by kcgen** [comment]
> What's the point of discussing how to create shortcuts if you don't even think the user should be creating them?

Shortcuts are a handy tip and I think it makes sense to contrast it with other recommended launch methods (frontend, command line, etc..).  So in that regard, I agree with @nemo93 that a couple examples on the tips and tricks page should have users covered. 

Shortcuts are highly nuanced per window manager: so I believe examples showing how the fields are populated is critical.

**Comment by es20490446e** [comment]
Okay, I will put it into trips and tricks.

**Comment by es20490446e** [comment]
I have simplified the section "[Shortcuts](https://github.com/dosbox-staging/dosbox-staging/wiki/Tips-&-Tricks#shortcuts)" on "Tips & Tricks", keeping some of your additions.

These are some of the things I see as potential improvements:
- The section seems more understandable without **examples**. 
- And without explanations on how to create shortcuts on each **operating system**.
- Each tip and trick could be more understandable by having its **own page**, because they are too unrelated. Instead the **table of contents** could have a section called "trips and tricks".
- **Emojis** look casual, but they really help to visually differentiate parts.
